test/e2e/ingress.go

```diff
@@ -21,6 +21,10 @@ import (
 	"path/filepath"
 	"time"
 
+	legacyv1 "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
> isn't that v1 usually?

it is more confusing to read that way as you start adding more.  "Which v1" is a question I don't want to mess with.